logstash/tests/logstash_test.py

```diff
@@ -622,6 +622,21 @@ def test_adding_in_pipeline():
     )
 
 
+def test_adding_in_pipeline_from_existing_configmap():
+    config = """
+existingPipelineConfigmap: uptime-pipeline
+"""
+    r = helm_template(config)
+
+    s = r["statefulset"][name]["spec"]["template"]["spec"]
+
+    assert {"configMap": {"name": "uptime-pipeline"}, "name": "logstashpipeline"} in s[
+        "volumes"
+    ]
+
+    assert r["statefulset"][name]["spec"]["template"]["metadata"]["annotations"] == None
```

Comment:
This assertion doesn't test the pipeline from configmap feature and so should be removed